db/version_set.h

```diff
@@ -1741,7 +1741,8 @@ class ReactiveVersionSet : public VersionSet {
       InstrumentedMutex* mu,
       std::unique_ptr<log::FragmentBufferedReader>* manifest_reader,
       Status* manifest_read_status,
-      std::unordered_set<ColumnFamilyData*>* cfds_changed);
+      std::unordered_set<ColumnFamilyData*>* cfds_changed,
+      std::vector<std::string>* files_to_delete);
```

Comment:
nit: since deletion is an action that is only applicable to the follower instance (as opposed to a secondary instance that also use this logic), would it make sense to call this something like `intermediate_files`.